docs/custom-flows/email-sms-otp.mdx

```diff
@@ -1,85 +1,67 @@
-</Callout>
-
-<Callout type="info">
-  Looking for two-factor authentication (2FA)? Check out our [Multi-factor authentication](/docs/custom-flows/mfa) guide.
+<Callout type="warning">
+  This guide is for users who want to build a *custom* user interface using the Clerk API. To authenticate users with one-time passwords using a *prebuilt* UI, you should use Clerk's [Account Portal pages](/docs/account-portal/overview) or [prebuilt components](/docs/components/overview).
 </Callout>
 
-## Before you start
-
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
-- You need to install the correct SDK for your application. You can find steps on how to do so through Clerk's [quickstart](/docs/quickstarts/overview) guides.
-
-## Set up passwordless authentication for your Clerk application
+Clerk supports passwordless authentication, which lets users sign in and sign up without having to remember a password. Instead, users receive a one-time password (OTP), also known as a one-time code, via email or SMS, which they can use to authenticate themselves.
 
-To enable passwordless authentication, you need to configure your instance to allow passwordless sign-ins. You can do this through the Clerk Dashboard or by creating a custom flow using the Clerk API.
+This guide will walk you through how to build a custom SMS OTP sign-up and sign-in flow. If you would like to use email OTP instead, the process is similar, and the differences will get pointed out along the way.
 
-### Enable passwordless authentication in the Clerk Dashboard
+{/* TODO: Update these Steps when the Steps component can accept other headings. As of right now, Steps can only accept H3s. */}
 
-Passwordless authentication can be configured through the Clerk Dashboard. Go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**. In the **Authentication strategies** section of this page, choose one of the available passwordless authentication strategies that send one-time codes.
+<Steps>
 
-For more information on how to configure passwordless authentication, check out the [Sign-in and sign-up options](/docs/authentication/configuration/sign-up-sign-in-options#authentication-strategies) guide.
+### Enable SMS OTP
 
-### Enable passwordless authentication using the Clerk API
+To use SMS OTP as an authentication strategy, you first need to enable it in the Clerk Dashboard.
 
-In case one of the above integration methods doesn't cover your needs, you can make use of lower-level commands and create a completely custom passwordless authentication flow.
+1. Navigate to the Clerk Dashboard.
+2. Go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
```

Comment:
```suggestion
2. Go to **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
```
![image](https://github.com/clerk/clerk-docs/assets/5156915/cc2fb09c-38db-43ef-bc8a-654c11e225c2)

Please also make this change in:
- https://clerk.com/docs/custom-flows/email-password
- https://clerk.com/docs/custom-flows/email-links